CHANGELOG.md

```diff
@@ -1,16 +1,21 @@
 ## Unreleased
 
+CHANGES:
+
+* `-write-secrets` flag now defaults to `false`, delegating file writes to the driver. [[GH-127](https://github.com/hashicorp/vault-csi-provider/pull/127)]
+  * **Note:** `-write-secrets` is deprecated and will be removed in the next major version.
+
 ## 0.3.0 (June 7th, 2021)
 
 FEATURES:
 
 * Support for changing the default Vault address and Kubernetes mount path via CLI flag to the vault-csi-provider binary [[GH-96](https://github.com/hashicorp/vault-csi-provider/pull/96)]
-* Support for sending secret contents to driver for writing via `--write-secrets=false` [[GH-89](https://github.com/hashicorp/vault-csi-provider/pull/89)]
-  * **Note:** `--write-secrets=false` will become the default from v0.4.0 and require secrets-store-csi-driver v0.0.21+
+* Support for sending secret contents to driver for writing via `-write-secrets=false` [[GH-89](https://github.com/hashicorp/vault-csi-provider/pull/89)]
+  * **Note:** `-write-secrets=false` will become the default from v0.4.0 and require secrets-store-csi-driver v0.0.21+
```

Comment:
Sorry I should have explained this change. With the standard lib `"flag"` package we're using, both forms work, but if you run with `-help` or `--help`, it prints the flags with a single dash, so I thought it would be better to be consistent on single dashes, which also matches better with the rest of our CLIs anyway.